packages/react-ui-components/.storybook/webpack.config.js

```diff
@@ -14,6 +14,13 @@ module.exports = (storybookBaseConfig, configType) => {
         //
         module: {
             rules: [
+                {
+                    test: /\.tsx?$/,
+                    exclude: /(node_modules)/,
+                    use: [{
+                        loader: 'awesome-typescript-loader'
```

Comment:
That's why we need to update to webpack 4.0.
The @latest awesome-typescript-loader does not work with webpack <4.
I use an old version of it here.
Also these loaders expect TypeScript 2.7, I think, so as we use TypeScript 3.x it's even more important to upgrade.

I agree that we should use the same loader in both places, but for me it would be this one, as it's considered considerably faster.

I'll get to that when we upgraded to webpack 4. (Or do you want to use ts-loader now and then switch, after the webpack upgrade is done?)

Reply:
I am on it but did not find time to fix the last issue :(